jobqueue/jobqueue_test.go

```diff
@@ -203,3 +204,35 @@ func TestJobQueueBuffer(t *testing.T) {
 	// We expect all 4 jobs to eventually be processed
 	assert.Len(t, processed, 4, "all jobs should have processed")
 }
+
+func TestJobQueueStress(t *testing.T) {
```

Comment:
Think it's worth converting this to a benchmark?

Reply:
IMHO this test is more valuable to confirm correctness rather than performance, I don't think it makes sense for it to be a benchmark.